examples/asset/asset-example.ipynb

```diff
@@ -71,7 +71,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 3,
```

Comment:
what is the reason for these changes to execution_count?

Reply:
When I added the file a while back, I didn't check the counts and they were incorrect.
Because I'm editing this file now, I combined your task and the one from my backlog to fix the counts.